lib/rubocop/cop/rspec/expect_actual.rb

```diff
@@ -66,6 +67,8 @@ class ExpectActual < Base
 
         def on_send(node)
           expect_literal(node) do |actual, matcher, expected|
+            next if SKIPPED_MATCHERS.include?(matcher)
+
             add_offense(actual.source_range) do |corrector|
               next unless SUPPORTED_MATCHERS.include?(matcher)
```

Comment:
I was a bit confused that we’d need both “skipped matchers” and “supported matchers”. Perhaps the supported matchers should be renamed “correctable matchers”? 😅